server/filters/get-tasl-markup.js

```diff
@@ -0,0 +1,10 @@
+import getLicenseInfo from './get-license-info';
+
+export default function({title, author, source, license}) {
+  const titleMarkup = source ? `<a rel="cc:attributionURL" property="dc:title" href="${source}">${title || source}</a>. ` : `<span property="dc:title">${title}.</span> `;
+  const authorMarkup = author ? `Credit: <span property="cc:attributionName">${author}.</span> ` : '';
+  const licenseInfo = license && getLicenseInfo(license);
+  const licenseMarkup = licenseInfo ? `<a rel="license" href="${licenseInfo.url}">${licenseInfo.text}</a>.` : '';
```

Comment:
Could we do this in a nunjucks template or JSX?

Reply:
We don't use this anywhere else, and it would be nice to avoid.

Reply:
I feel it's cleaner here than clogging up the markup with conditionals.

Reply:
> clogging up the markup with conditionals

If I search for `{% if` I get 152 results in the markup.
Isn't conditionals what templating is for?

Reply:
within reason!

Reply:
Again, I really don't like using the another way of doing templating that is quite old school, but as it's my preference, let's get it in and I can use this as an example for using JSX.